src/shelter/shelter.ts

```diff
@@ -122,4 +160,39 @@ export class Shelter {
       switchMap(value => (value === undefined ? throwError('Failed to reveal private key') : of(value))),
       map(privateKey => new InMemorySigner(privateKey))
     );
+
+  static unlockAppWithBiometry$ = () =>
+    from(Shelter.biometryIsAvailable()).pipe(
+      switchMap(({ available, error }) => {
+        if (!available) {
+          return throwError(error);
+        }
+
+        return from(ReactNativeBiometrics.biometricKeysExist());
+      }),
+      switchMap(({ keysExist }) => {
+        if (!keysExist) {
+          throw new Error("Biometry keys don't exist");
+        }
+
+        return from(
+          ReactNativeBiometrics.createSignature({
```

Comment:
Actually, we can do without this request if the default text of prompt satisfies requirements